Dockerfile.base

```diff
@@ -31,6 +31,10 @@ RUN apt-get update && \
     wget \
     dirmngr \
     openssh-client \
+    locales \
+  && dpkg-reconfigure locales \ 
+  && locale-gen en_US.UTF-8 \
+  && export LANG=en_US.UTF-8 LANGUAGE=en_US:en LC_ALL=en_US.UTF-8 \
```

Comment:
these are lost on the running container, but i dont think theyre needed with the other 2 lines

Reply:
Yeah. Just double checking we remove this line and use `-e LC_ALL=en_US.UTF-8` when running the container?

Reply:
I dont think so, but I'm unsure